packages/hardhat-plugin/src/index.ts

```diff
@@ -227,55 +232,106 @@ task("visualize")
     }
   );
 
-task("ignition-info")
-  .addParam("deploymentId")
-  .addFlag("json", "format as json")
+task("status")
+  .addParam("id")
   .setDescription("Lists the deployed contract addresses of a deployment")
-  .setAction(
-    async (
-      {
-        deploymentId,
-        json: formatAsJson,
-      }: { deploymentId: string; json: boolean },
-      hre
-    ) => {
-      const deploymentDir = path.join(
-        hre.config.paths.ignition,
-        "deployments",
-        deploymentId
+  .setAction(async ({ id }: { id: string }, hre) => {
+    const deploymentDir = path.join(
+      hre.config.paths.ignition,
+      "deployments",
+      id
+    );
+
+    const statusResult = await status(deploymentDir);
+
+    if (statusResult.started.length > 0) {
+      console.log("");
+      console.log(
+        chalk.bold(`⛔ Deployment ${id} exited early and needs to be rerun`)
```

Comment:
If there is a simulation error, we don't update the future state, we kill the process leaving started futures.

If you call status on a deployment after the simulation error you will see this message.

Reply:
How is this different from the `"There are futures that were started but have not completed:"`?

Reply:
What I mean is that they feel duplicated

Reply:
That is interesting. I was trying to sign post the user in the case where there was a simulation error; which I think is likely to be the most common case here. `There are futures that were started but have not completed`, is accurate but obscure for that case is my suspicion.

Reply:
I have updated the text and combined the two separate lines into one.